package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "lambda-cloudwatch-slack",
-  "version": "0.2.1",
+  "version": "0.3.0",
   "description": "Better Slack notifications for AWS CloudWatch",
   "authors": [
     "Assertible <admin@assertible.com>",
     "Christopher Reichert <creichert07@gmail.com>",
-    "Cody Reichert <codyreichert@gmail.com>"
+    "Cody Reichert <codyreichert@gmail.com>",
+    "Alexandr Promakh <s-promakh@ya.ru>"
```

Comment:
+1. Can you do me a favor? And _remove_ assertible? I can do it in a follow-up also, no worries.

Reply:
No problem, I will do it.

Reply:
thank you